src/router/index.js

```diff
@@ -12,6 +14,16 @@ const routes = [
     path: '/pedidos',
     name: 'pedidos',
     component: Pedidos
+  },
+  {
+    path: '/cadastro',
+    name: 'cadastro',
+    component: Cadastro
+  },
+  {
+    path: '/login',
```

Comment:
Your routes are defined in lowercase, but the use of them with a another case

Reply:
Ajusted.